.github/workflows/gh-deploy-production.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy to staging and preview deploys
```

Comment:
Should say it's deploying to prod, right?

Reply:
correct, ty!